pvlib/test/test_pvsystem.py

```diff
@@ -383,6 +384,27 @@ def test_PVSystem_calcparams_desoto(cec_module_params):
     assert_allclose(nNsVth, 0.473)
 
 
+def test_sdm_sum_current():
```

Comment:
I vote for putting this in a new ``test_gold_sdm.py`` module. And the import above.